cmd/appengine/device.go

```diff
@@ -571,6 +581,9 @@ func devicesDataSnapshotF(command *cobra.Command, args []string) error {
 				if err != nil {
 					warnOrFail(snapshotInterface, i.Name, err)
 				}
+
+				utils.MaybeCurl(snapshotCall, astarteAPIClient)
```

Comment:
Not sure if we want to print the equivalent curl **and** run the request.
We should consider a more robust approach